fiaas_deploy_daemon/deployer/kubernetes/deployment/deployer.py

```diff
@@ -244,6 +245,14 @@ def _build_fiaas_env(config):
     return env
 
 
+def _add_config_maps(app_spec):
+    config_maps = [EnvFromSource(configMapRef=ConfigMapEnvSource(name=app_spec.name, optional=True))]
```

Comment:
Does this mean that the same env vars defined in one of the extra configmaps will overwrite those from this one, as they're added afterwards? To me, it seems like the configmap named for the app should be the most specific. In either case, I think it should be documented.

Reply:
I'm not sure, I have to do some research to see what happens when it overwrites. If anyone have a good way to test this I'm all ears.

Reply:
@mortenlj Do you know how that works? Also, what happens today if you create a configMap with env-vars already defined within the cluster?

Reply:
Found this in the documentation for the `envFrom` field:

> When a key exists in multiple sources, the value associated with the last source will take precedence. Values defined by an Env with a duplicate key will take precedence.